pkglistgen/tool.py

```diff
@@ -681,6 +730,21 @@ def update_and_solve_target(
         file_utils.copy_directory_contents(group_dir, product_dir, ignore_list)
         file_utils.change_extension(product_dir, '.spec.in', '.spec')
         file_utils.change_extension(product_dir, '.product.in', '.product')
+        if productcompose_dir:
```

Comment:
FWICT this is effectively always true. Meant to be `os.path.isdir(productcompose_dir`?

Reply:
atm I want it always to create a local version, even when there is no server side package created yet.
Yes, we should change this later, when we drop the old code.

Reply:
Ok, so this is a hack for development.

Let's keep this thread open so it won't be forgotten.